docs/release/release_0_3_5.md

```diff
@@ -0,0 +1,66 @@
+# napari 0.3.5
+
+We're happy to announce the release of napari 0.3.5!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+This release contains a number of bug fixes on various platforms. For those
+interested in napari performance, we have added a new performance monitoring
+mode, that can be activated by the `NAPARI_PERFMON` environment variable, see
+(#1262) for details. We have also added a page on napari's rendering pipeline
+in (#1328).
+
+
+## New Features
+- Allow using of custom color dictionary in labels layer (#1339 and #1362)
+- Add performance monitoring widget (#1262)
```

Comment:
Is #1342 being included in this release? If so, should we add it here?

```suggestion
- Add performance monitoring widget (#1262)
- Allow Shapes face and edge colors to be mapped to properties (#1342)
```

Reply:
yes, not sure why that wasn't there, i guess i made release notes before merge or something. thanks for catching!!